share/spice/sports/nfl/games/sports_nfl_games.js

```diff
@@ -65,6 +65,11 @@
                     // fill boxscore with placeholder data
                     attrs.score = games.fillBoxscore(attrs.score, ops);
 
+                    // finish here if there is no possession data
+                    if (!attrs.score.possession) {
```

Comment:
@b1ake given the definition of [`fillBoxscore`](https://github.com/duckduckgo/zeroclickinfo-spice/blob/04aaf11a04664c390accb8d5676618eacfa4b950/share/spice/sports/sports.js#L207) it seems that it easily might be.

Will update the PR, thanks for catching it!